app/assets/javascripts/error_handler.coffee

```diff
@@ -0,0 +1,23 @@
+email = 'support@roqua.nl'
+message = "Onbekende fout. Mail naar <a href='#{email}'>#{email}</a> voor hulp"
```

Comment:
Op zich zou dat goed zijn ja, alleen ben ik bang dat er toch gevallen zijn waar we niet genoeg informatie hebben op basis van de exceptie alleen.

Hiermee breng je me trouwens op het idee om de hele store mee te sturen met de exceptie 👍